site/guide/manage-permissions.qmd

```diff
@@ -0,0 +1,47 @@
+---
+title: "Manage permissions"
+---
+
+These steps guide you through the process of configuring and overseeing permissions in the {{< var vm_platform >}}. By carefully defining what users can and cannot do, you can ensure that the right individuals have the appropriate level of access to resources and can perform the correct actions. 
+
+## Prerequisites
+
+To perform these steps, you must hold the Customer Admin role. 
+
+<!-- We can't add roles yet, feature not implemented -->
+
+<!-- ## Add roles
+
+To add a new role:
+
+1. [Log in to the {{< var vm_platform >}}]({{< var vm_url >}}).
+
+1. Click **Settings**.
+
+{{< include _manage-roles-and-permissions-roles.qmd >}} -->
+
+## Steps
+
+1. [Log in to the {{< var vm_platform >}}]({{< var vm_url >}}).
+
+1. Click **Settings**.
+
+{{< include _manage-permissions.qmd >}}
+
+::: {.callout-important}
+Be careful when modifying permissions to ensure they align with the role's intended responsibilities and access requirements.
+:::
+
+<!---
+## Troubleshooting
+
+[Include any common issues or errors that may arise during the task and how to resolve them.]
+
+
+[Summarize the task and provide any next steps or resources for the user to continue their learning or work.]
+--->
+
+## What's next
```

Comment:
As a comment on both "Manage roles" and "Manage permissions", I'm not sure these related links add value — I would just remove. We already get some of this for free via the bottom links (see the screenshot) and these seem like parallel rather than sequential tasks. 

<img width="1037" alt="image" src="https://github.com/validmind/documentation/assets/15148011/3932b6ff-7f12-4724-b64a-7c3bcc805016">